easybuild/framework/easyconfig/default.py

```diff
@@ -165,6 +165,7 @@
 
     # OTHER easyconfig parameters
     'buildstats': [None, "A list of dicts with build statistics", OTHER],
+    'options' : [{},"A list of extra options for specific blocks", OTHER],
```

Comment:
currently, it's exactly that, 'pass anything to the block'.

It's needed for the `modulename` but maybe should look to a cleaner solution for that.